efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -406,7 +389,7 @@ def __init__(self,
 
       bn_per_level = {}
       for level in range(self.min_level, self.max_level + 1):
-        bn_per_level[level] = utils_keras.build_batch_norm(
+        bn_per_level[str(level)] = utils_keras.build_batch_norm(
```

Comment:
Why I need this?

Reply:
This change allows me to save the model, I found this trick here: https://github.com/tensorflow/tensorflow/issues/36916#issuecomment-640395999

P.S. I mentioned it in commit message.

Reply:
Change dict to list should fix the bug?

Reply:
yes, but there he explains why to touch `__dict__` is problematic.

Reply:
> yes, but there he explains why to touch `__dict__` is problematic.

Good Observation.